### PR TITLE
Jakarta migration - fix FW injection by migration remaining imports

### PR DESCRIPTION
**Pull request by michalvavrik**
### Summary

https://github.com/quarkus-qe/quarkus-test-framework/pull/668 follow-up where I missed javax Inject. Imports are sorted as CI forced me to.

Please check the relevant options

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update
- [x] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [X] Example scenarios has been updated / added
- [X] Methods and classes used in PR scenarios are meaningful
- [X] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by michalvavrik** [comment]
Blocks https://github.com/quarkus-qe/quarkus-test-suite/pull/1049

**Comment by michalvavrik** [comment]
run tests

**Comment by michalvavrik** [comment]
run tests

File: quarkus-test-openshift/src/main/java/io/quarkus/test/services/quarkus/OpenShiftS2iGitRepositoryQuarkusApplicationManagedResource.java [hunk 1]
```diff
@@ -23,9 +23,8 @@
     private static final String QUARKUS_SOURCE_S2I_SETTINGS_MVN_FILENAME = "settings-mvn.yml";
     private static final String INTERNAL_MAVEN_REPOSITORY_PROPERTY = "${internal.s2i.maven.remote.repository}";
     private static final PropertyLookup MAVEN_REMOTE_REPOSITORY = new PropertyLookup("s2i.maven.remote.repository");
-    // TODO: use ubi-quarkus-native-binary-s2i once upstream https://github.com/quarkusio/quarkus/issues/30829 is fixed
     private static final PropertyLookup QUARKUS_NATIVE_S2I_FROM_SRC = new PropertyLookup("s2i.openshift.base-native-image",
-            "quay.io/quarkus/ubi-quarkus-native-s2i:22.2-java17");
+            "quay.io/quarkus/ubi-quarkus-graalvmce-s2i:22.3-java17");
```
Comment (fedinskiy) [inline quarkus-test-openshift/src/main/java/io/quarkus/test/services/quarkus/OpenShiftS2iGitRepositoryQuarkusApplicationManagedResource.java:27]: I remember there were some complication, about proper usage of images. Is it the correct one?

And this change is neither in the PR description nor in the text of the commit.

**Review by fedinskiy — CHANGES_REQUESTED** [review]
And please check the CI

File: quarkus-test-openshift/src/main/java/io/quarkus/test/services/quarkus/OpenShiftS2iGitRepositoryQuarkusApplicationManagedResource.java (hunk 1 above)
Comment (michalvavrik) [reply quarkus-test-openshift/src/main/java/io/quarkus/test/services/quarkus/OpenShiftS2iGitRepositoryQuarkusApplicationManagedResource.java:27]: You are correct, this change is in this commit by mistake (good catch btw!! I was trying to test whether it works). However strictly speaking it is according to what should be there as you can read here https://github.com/quarkusio/quarkus/issues/30829#issuecomment-1422548366. Currently all related tests are (and going to be in TS too) disabled anyway, there is still TODO. Please let me correct comment in separate issue as this one is rather urgent (it blocks me in TS). But I'll respect any preference you have to fix this...

**Comment by michalvavrik** [comment]
I'll check CI ASAP and get back here.

**Comment by michalvavrik** [comment]
OC native failure is due to _There is insufficient memory for the Java Runtime Environment to continue._, IMHO not related.

File: quarkus-test-openshift/src/main/java/io/quarkus/test/services/quarkus/OpenShiftS2iGitRepositoryQuarkusApplicationManagedResource.java (hunk 1 above)
Comment (michalvavrik) [reply quarkus-test-openshift/src/main/java/io/quarkus/test/services/quarkus/OpenShiftS2iGitRepositoryQuarkusApplicationManagedResource.java]: got rid of it

**Review by fedinskiy — APPROVED** [review]

**Pull request merged**